rosbag2_performance/rosbag2_benchmarking/scripts/install_pkexec_iotop.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+echo "--------------"
+echo "This script copies iotop policy to '/usr/share/polkit-1/actions/iotop.policy' so it will always be allowed to with root privileges."
+echo ""
+echo "File contents: "
+echo ""
+cat $SCRIPTPATH/iotop.policy
+echo ""
+echo "--------------"
+
+sudo cp $SCRIPTPATH/iotop.policy /usr/share/polkit-1/actions/iotop.policy
```

Comment:
👎  I don't appreciate that we modify users configs here.

Reply:
Yes, indeed we included an `iotop` custom policy here and modifying anything in `usr/share` is controversial at best. However, using this policy is not necessary. It is just a noticeable quality of live improvement for large test bundles. User can still enter password before each test and everything will be fine.

Sudo is required because of the system utilization tool we are using (`iotop`) and this is reasonable because we are literally recording system stats at the time.

We don’t really appreciate that as well so we will look for an different tool which will not require root privileges. Alternatively we can think about moving out the `iotop` instance from single benchmark to something that will run more globally and require only one sudo prompt. I will look into that!